packages/components/package.json

```
@@ -20,17 +20,20 @@
     "build": "concurrently 'npm:build:*'",
     "build:js": "rollup --config",
     "build:types": "glint --declaration",
-    "prepublishOnly": "yarn workspace @hashicorp/ember-flight-icons build && yarn build && test -f 'dist/styles/@hashicorp/design-system-components.css' || (echo 'The pre-compiled CSS file was not found' && exit 1)",
-    "start": "rollup --config --watch --environment development",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint": "concurrently 'yarn:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'yarn:lint:*:fix' --names 'fix:'",
     "lint:css": "stylelint \"src/styles/**/*.scss\"",
-    "lint:css:fix": "npm-run-all \"lint:css --fix\"",
-    "lint:types": "glint",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:css:fix": "stylelint \"src/styles/**/*.scss\" --fix",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix"
+    "lint:js:fix": "eslint . --fix",
+    "lint:types": "glint",
+    "prepublishOnly": "yarn workspace @hashicorp/ember-flight-icons build && yarn build && test -f 'dist/styles/@hashicorp/design-system-components.css' || (echo 'The pre-compiled CSS file was not found' && exit 1)",
+    "start": "rollup --config --watch --environment development",
+    "start:js": "rollup --config --watch --no-watch.clearScreen",
+    "start:types": "glint --declaration --watch",
```

In the other `package.json` files we have only the `start` command. Are the other two needed? What are they used for? Doesn't the `start` command do the same (only a bit more)?
```suggestion
    "start": "rollup --config --watch --environment development",
```

---

I guess it just a convenience command that got just introduced in the latest blueprint, happy drop this changes is there is no value

---

[components](https://github.com/hashicorp/design-system/blob/a3c3f3da5abea4fecdce9d0d190562aa9241664c/packages/components/package.json#L34) packages has similar commands but they are bypassed for some reason, happy to drop this change if you see no value

---

@aklkv we discussed that these changes would be removed or moved to https://github.com/hashicorp/design-system/pull/2142. Did I misunderstand?